go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	cloud.google.com/go/compute v1.5.0
 	cloud.google.com/go/iam v0.3.0 // indirect
-	github.com/hashicorp/go-getter v1.5.11
+	github.com/hashicorp/go-getter v1.6.1
```

Comment:
IIUC this is used to fetch terraform modules from GitHub. Since we do not yet have an integration test that performs such an action (daos hopefully coming soon), we should probably manually test that it still work.

Reply:
`make tests` covers `ghcp create` of the DAOS examples which relies upon go-getter to determine the input variables accepted by the DAOS modules. But, yes, first thing I did was manually run it on a DAOS example.